### PR TITLE
Add default timeout

### PR DESCRIPTION
**Pull request by TinOo512**
Hi, 

First, thanks for Labbable we love it here for our test :)

I wanted to override the default timeout of 2000ms.
I'm not sure about the constructor signature, let me know if you prefer something more like : 

``` js
constructor(server, options)
```

I'll edit the readme if this sound good for you and you are okay to merge.

### CODE REVIEW

**Comment by devinivy** [comment]
I can get down with that!  Just so you know, this will not be a backwards-compatible change, which I'm okay with.  I will also likely remove the plugin from the next version and fix some minor timing oddities.

File: test/index.js [hunk 1]
```diff
@@ -228,7 +228,40 @@ describe('Labbable', () => {
...
+                expect(err.message).to.equal('Labbable timed-out after 1000ms.  Did you forget to call labbable.using(server)?');
+                expect(srv).to.not.exist();
+                expect(called.length).to.equal(1);
+                expect(called[0]).to.equal(1000);
+                done();
+            });
+
+        });
+
+        it('has default timeout configurable in .', (done, onCleanup) => {
```
Comment (devinivy) [inline test/index.js]: Can you fix this test description?

Comment (TinOo512) [reply test/index.js]: done

**Comment by TinOo512** [comment]
@devinivy I don't really see the value of `labbable` when `Glue` isn't used and the server can be accessed synchronously. So it's not a problem for me to remove the plugin :)

I fixed the test description and edited the `README`, what do you think ?

**Comment by TinOo512** [comment]
@devinivy Do you plan to release anytime soon ? Thx anyway !

**Comment by devinivy** [comment]
This is on my radar– sorry for the delay.  This PR is fine as-is 👍

**Comment by devinivy** [comment]
Published in v2 alongside some minor timing fixes– thank you for your patience!

**Pull request merged**